libraries/radspreadstreamprocessing/features/page-setup-exportedr.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Page Setup Exporter
+page_title: Page Setup Exporter
+slug: radspreadstreamprocessing-features-page-setup-exporter
+tags: worksheet, page, exporter
+published: True
+position: 3
+---
+
+# Page Setup Exporter
+
+The __PageSetupExporter__ allows you to export the page settings for printing. The following methods are exposed:
+
+* __SetFitToPagesTall:__ Sets the number of pages tall the worksheet will be scaled to when it's printed.
+* __SetFitToPagesWide:__ Sets the number of pages wide the worksheet will be scaled to when it's printed.
+* __SetPageOrder:__ Sets the page order.
+* __SetPageOrientation:__ Sets the page orientation.
+* __SetPaperSize:__ Sets the size of the paper.
+* __SetScaleFactor:__ Sets the scale factor of the printed worksheet. The valid values are from 0.1 to 4.
+
+
+## Working with PageSetupExporter
+
+Importan part is that you need to place the __PageSetupExporer__ after the code for exporting all cells on the sheet. Example 1 demostrates how you can create the __PageSetupExporter__ and where to palce it. 
```

Comment:
An important part is that you need to place the __PageSetupExporer__ after the code for exporting all cells on the sheet. Example 1 demonstrates how you can create the __PageSetupExporter__ and where to place it.